src/js/vendor/wasm_exec.js

```
@@ -208,10 +216,16 @@
 				return decoder.decode(new DataView(this._inst.exports.mem.buffer, saddr, len));
 			}
 
+			const testCallExport = (a, b) => {
```

ESLint

[error] Unexpected tab character.

(no-tabs)